galasa-extensions-parent/dev.galasa.ras.couchdb/src/main/java/dev/galasa/ras/couchdb/internal/CouchdbValidatorImpl.java

```diff
@@ -76,16 +78,16 @@ public void checkCouchdbDatabaseIsValid( URI rasUri, CloseableHttpClient httpCli
             checkIndex(httpClient, rasUri, 1, "galasa_run", "result");
 
             logger.debug("RAS CouchDB at " + rasUri.toString() + " validated");
-        } catch (CouchdbRasException e) {
+        } catch (CouchdbException e) {
             throw e;
         } catch (Exception e) {
-            throw new CouchdbRasException("Validation failed", e);
+            throw new CouchdbException("Validation failed "+ e);
```

Comment:
Normally we would only throw an exception which we wrote the code for. Using someone else's exception type is a bit dodgy.

Reply:
OK. I get it now. I see that it is one of ours, just made not RAS specific.